### PR TITLE
Updated Readme.md

### PR DESCRIPTION
**Pull request by Noobham**
Fixes https://github.com/plone/volto/issues/5389

- Documentation Update
- Updated required version of npm(node package manager) in README.md.

I, Shubham Kumar, agree to have this contribution published under Creative Commons 4.0 International License (CC BY 4.0), with attribution to the Plone Foundation.

### CODE REVIEW

File: news/5389.documentation [hunk 1]
```diff
@@ -0,0 +1 @@
+Updated npm version required for volto.org . @Noobham
```
Comment (stevepiercy) [inline news/5389.documentation:1]: npm is not Node.js. Where did you get `volto.org`?

```suggestion
Updated Node.js version required for Volto. @Noobham
```

File: README.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ For links to trainings and videos, see [Other learning resources](https://6.docs
 ## Supported Node.js versions
 
 Volto runs using [Node.js](https://nodejs.org/).
-We recommend using the current LTS version (18.x).
+We recommend using the current LTS version.
 
 - Node.js 20: Supported since Volto 17.
```
Comment (stevepiercy) [inline README.md:50]:
```suggestion
-   Node.js 20 LTS: supported since Volto 17.
```

File: README.md [hunk 3]
```diff
@@ -45,7 +45,7 @@ For links to trainings and videos, see [Other learning resources](https://6.docs
...
 - Node.js 18 (LTS): Supported since Volto 17.
```
Comment (stevepiercy) [inline README.md:51]:
```suggestion
-   Node.js 18 LTS: supported since Volto 17.
```

File: README.md [hunk 4]
```diff
@@ -45,7 +45,7 @@ For links to trainings and videos, see [Other learning resources](https://6.docs
 ## Supported Node.js versions
 
 Volto runs using [Node.js](https://nodejs.org/).
-We recommend using the current LTS version (18.x).
+We recommend using the current LTS version.
```
Comment (stevepiercy) [inline README.md:48]:
```suggestion
We recommend using the most recent LTS version that Volto supports.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Review by stevepiercy — APPROVED** [review]
@sneridagh this is ready for review and merge.

**Comment by stevepiercy** [comment]
I don't know why the two CI jobs fail. They are totally unrelated to this PR.

**Comment by stevepiercy** [comment]
CI passes now. @sneridagh

**Review by sneridagh — APPROVED** [review]

**Pull request merged**